lib/Github/Api/Migrations/SourceImport.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Github\Api\Migrations;
+
+use Github\Api\AbstractApi;
+use Github\Api\AcceptHeaderTrait;
+use Github\Exception\InvalidArgumentException;
+use Github\Exception\MissingArgumentException;
+
+/**
+ * Support for Migrations.
+ *
+ * @link   http://developer.github.com/v3/migrations
+ *
+ * @author Sebastian Berm <sebastian@sebsoft.nl>
```

Comment:
Also add the `@experimental` annotation, to indicate this is a preview endpoint. See #806